interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName+".bpf")
```

Comment:
If the file names are being extended with .bpf, is there any benefit in changing SnapSeccompDir to have .bpf in the directory itself, or could we simply keep the same directory and have .bpf in the filenames alone?  This has the added benefit that the old profiles will be properly collected once new profiles compile, instead of being left around forever once this patch lands.